terraform/modules/len/main.tf

```diff
@@ -42,9 +42,14 @@ resource "aws_iam_role" "len" {
   assume_role_policy = data.aws_iam_policy_document.len_assume_policy.json
 }
 
+resource "aws_iam_role_policy_attachment" "len_xray_access" {
+  role       = aws_iam_role.len.name
+  policy_arn = "arn:aws:iam::aws:policy/AWSXRayDaemonWriteAccess"
```

Comment:
This is currently
```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": [
                "xray:PutTraceSegments",
                "xray:PutTelemetryRecords",
                "xray:GetSamplingRules",
                "xray:GetSamplingTargets",
                "xray:GetSamplingStatisticSummaries"
            ],
            "Resource": [
                "*"
            ]
        }
    ]
}
```
per [here](https://docs.aws.amazon.com/lambda/latest/dg/lambda-intro-execution-role.html#:~:text=Select%20the%20AWS%20managed%20policies%20AWSLambdaBasicExecutionRole%20and%20AWSXRayDaemonWriteAccess.)